js/cards.js

```diff
@@ -48,6 +48,11 @@ const sort_functions = {
       }
     },
 };
+let card_count = 10;
```

Comment:
rename to something like `card_z_index` so it's clearer what this is for

Reply:
Update this name for clarity. Originally it was literally used for counting how many cards had been opened, haha. :)